src/config/rt.rs

```diff
@@ -101,10 +104,10 @@ pub struct RtcClean {
 }
 
 impl RtcClean {
-    pub(super) fn new(opts: ConfigOptsClean) -> Result<Self> {
-        Ok(Self {
+    pub(super) fn new(opts: ConfigOptsClean) -> Self {
```

Comment:
I changed this because clippy wanted me to. It's not an external API so it doesn't matter if it is changed again in the future

Reply:
Yea, that's totally fine. I probably originally made it return a result just because the others were doing that, or because I refactored the fallible code out at some point in the past. Either way, this is fine.